pkgs/applications/editors/neovim/tests/default.nix

```diff
@@ -55,9 +55,11 @@ let
   nvim-with-luasnip = wrapNeovim2 "-with-luasnip" (makeNeovimConfig {
     plugins = [ {
         plugin = vimPlugins.luasnip;
-
       }
     ];
+    doCheck = true;
+    # build should fail when this gets uncommented
+    # luaRcContent = "toto";
```

Comment:
Is this supposed to stay ? I see how it can be an example.

Reply:
I was not sure how to test for a build failure. So I added this to be able to check. Happy to remove it or you have another suggestion

Reply:
> I was not sure how to test for a build failure.

You can use [`pkgs.testers.testBuildFailure`](https://nixos.org/manual/nixpkgs/unstable/#tester-testBuildFailure) to make a test that passes when the derivation you pass it fails.